preview/preview.js

```diff
@@ -66,7 +66,9 @@
 			'OpenPtMap',
 			'OpenRailwayMap',
 			'OpenFireMap',
-			'SafeCast'
+			'SafeCast',
+			'TomTom.Hybrid',
+			'TomTom.Labels'
```

Comment:
Are you sure that the API key provided will handle the amount of hits on demo page ?

Reply:
Not sure. This key is from my personal Dev account with "2,500 TomTom Maps API transactions for free on a daily basis".

Reply:
The key may be used more than that and if it's the case the demo page will show an error about that, maybe it's more safe to not show these tiles on the demo page.

Reply:
Done